### PR TITLE
Fussing

### PR DESCRIPTION
**Pull request by mslinn**
Minor nitpicks: type attribution for public members, removing extra parentheses, etc

### CODE REVIEW

File: launcher/src/test/scala/LaunchConfigSpec.scala [hunk 1]
```diff
@@ -17,16 +17,16 @@ class LaunchConfigSpec extends Specification { def is = s2"""
 
   """
```
Comment (larsrh) [inline launcher/src/test/scala/LaunchConfigSpec.scala]: Please revert the changes in this file below. Those are just test functions.

**Review by larsrh — CHANGES_REQUESTED** [review]
Thanks for the PR! I only have one minor request, then this is good to merge.

**Comment by larsrh** [comment]
@mslinn Out of curiosity, are you planning to use this for something?

**Comment by mslinn** [comment]
The thought of using this project crossed my mind, but [I went native instead](https://github.com/mslinn/cli-loop/blob/master/build.sbt#L48).

**Comment by mslinn** [comment]
I find test functions helpful to learn how a project works. Knowing the types of variables assists the reader. Why revert?

**Comment by larsrh** [comment]
> Knowing the types of variables assists the reader.

Sure, but I firmly subscribe to the philosophy that I put type annotations on entities that are part of the public API. Tests are not public API. In particular, these functions return implementation details of specs2.

**Comment by mslinn** [comment]
I comply (*beep*).

**Comment by larsrh** [comment]
Thanks!

**Comment by larsrh** [comment]
Somewhat unrelatedly: I saw that you also have JRuby and Jython on the same classpath. This might cause problems under some circumstances, see #3.

**Comment by mslinn** [comment]
Thanks, Lars, I did not know of the problem or the preferred solution: `jython-shaded` 👍
